src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsResourceHearingsIntTest.kt

```diff
@@ -470,4 +470,144 @@ class AdjudicationsResourceHearingsIntTest : IntegrationTestBase() {
       }
     }
   }
+
+  @DisplayName("DELETE /adjudications/adjudication-number/{adjudicationNumber}/hearings/hearingId")
+  @Nested
+  inner class DeleteHearing {
+    private val offenderNo = "A1965NM"
+    private lateinit var prisoner: Offender
+    private lateinit var reportingStaff: Staff
+    private lateinit var existingIncident: AdjudicationIncident
+    private lateinit var existingHearing: AdjudicationHearing
+    private val existingAdjudicationNumber = 123456L
+
+    @BeforeEach
+    fun createPrisonerAndAdjudication() {
+      nomisDataBuilder.build {
+        reportingStaff = staff(firstName = "JANE", lastName = "STAFF") {
+          account(username = "JANESTAFF")
+        }
+        existingIncident = adjudicationIncident(reportingStaff = reportingStaff) {}
+        prisoner = offender(nomsId = offenderNo) {
+          booking {
+            adjudicationParty(incident = existingIncident, adjudicationNumber = existingAdjudicationNumber) {
+              val previousCharge = charge(offenceCode = "51:1B")
+              existingHearing = hearing(
+                internalLocationId = aLocationInMoorland,
+                scheduleDate = LocalDate.parse("2023-01-02"),
+                scheduleTime = LocalDateTime.parse("2023-01-02T14:00:00"),
+                hearingDate = LocalDate.parse("2023-01-03"),
+                hearingTime = LocalDateTime.parse("2023-01-03T15:00:00"),
+                hearingStaff = reportingStaff,
+                hearingTypeCode = AdjudicationHearingType.GOVERNORS_HEARING,
+              ){
+                result(
+                  charge = previousCharge,
+                  pleaFindingCode = "NOT_GUILTY",
+                  findingCode = "PROVED",
+                ) {
+                  award(
+                    statusCode = "SUSPENDED",
+                    sanctionCode = "ADA",
+                    effectiveDate = LocalDate.parse("2023-01-08"),
+                    statusDate = LocalDate.parse("2023-01-09"),
+                    sanctionDays = 4,
+                    sanctionIndex = 1,
+                  )
+                }
+                notification(staff = reportingStaff, deliveryDate = LocalDate.parse("2023-01-04"), deliveryDateTime = LocalDateTime.parse("2023-01-04T10:00:00"))
+              }
```

Comment:
Adding Hearings, results, awards and notifications to test hearing delete cascade